app/controllers/todos_controller.rb

```diff
@@ -1,7 +1,21 @@
 class TodosController < ApplicationController
+    @@sort_Type = "none"
     before_action :authenticate
     def index
-        @todos = current_user.todos
+        if params[:sort]
```

Comment:
What is the point of this if statement?

Reply:
This is an error prevention if statement. Without this if statement, if a user was to mark a todo complete or incomplete then the website will crash as without it the code will assume there will be a change in the sort.